src/OrbitGl/TimerChain.h

```diff
@@ -14,7 +14,7 @@
 #include "OrbitBase/Logging.h"
 #include "TextBox.h"
 
-static constexpr int kBlockSize = 1024;
+static constexpr int kBlockSize = 64 * 1024;
```

Comment:
Just one note that, if I remember correctly, in some places (some `UpdatePrimitives` I guess) we go through the blocks and, based on `min_timestamp_` and `max_timestamp_`, we either skip the content of the block completely or go through each the `TextBox` in the block to see whether they are in the time range we are interested in. This might have no actual effect on that or even be an optimization, but it affects more than just memory allocation.

Reply:
Good point, thanks, I'm reverting the change in block size for now to be sure.